### DISCUSSION TITLE
Extensions not working, random freezing on open (Linux Mint 20.2, ublock origin/keepassxc-browser)

### DISCUSSION THREAD

**Discussion by RandomGHUser** [Eloston/ungoogled-chromium#1769] [General]
## Base 
*  Fresh install of Linux Mint 20.2, install ungoogled-chromium (current repo version 95.0.4638.54)
  ```sh
  echo 'deb http://download.opensuse.org/repositories/home:/ungoogled_chromium/Ubuntu_Focal/ /' | sudo tee /etc/apt/sources.list.d/home-ungoogled_chromium.list > /dev/null
  curl -s 'https://download.opensuse.org/repositories/home:/ungoogled_chromium/Ubuntu_Focal/Release.key' | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home-ungoogled_chromium.gpg > /dev/null
  sudo apt update
  sudo apt install -y ungoogled-chromium
  ```
* Install the store
```sh
wget https://github.com/NeverDecaf/chromium-web-store/releases/download/v1.4.3/Chromium.Web.Store.crx
#open chrome://extensions, enable dev mode, drop crx in, it installs
```
### Ublock origin
* downloading the UBlock origin crx 
```sh
# https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm
# drop into chromium
```
I can not access ANY website after the plugin is enabled.  If I disable the plugin, all websites work normally

### KeepassXC-Browser
* download the package
```sh
wget https://github.com/keepassxreboot/keepassxc-browser/releases/download/1.7.11/keepassxc-browser_1.7.11_chromium.zip
# open chrome://extensions, enable dev mode, drop zip in, it installs
```
KeepassXC-Browser will not connect to any of my open KeepassXC databases (and the chromium browser integration is enabled, verified under NativeMessagingHosts.

## Random freezes when opening
Sometimes when opening, I only see the chromium titlebar, and the application never fully opens.

## History

Everything had been working up until possibly a couple weeks ago on my other laptops (upgraded to LM 20.2 / chromium update)
I have one laptop where the browser and plugins are properly working (Linux Mint 20.1 / Chromium 90.0.4430.212).  I've tried d/ling the older version of Chromium on LM 20.2, but still, random freezing and extensions aren't working.

**Comment by networkException** [comment]
This is a duplicate of https://github.com/ungoogled-software/ungoogled-chromium-debian/issues/247, please use the workaround described in https://github.com/ungoogled-software/ungoogled-chromium-debian/issues/247#issuecomment-950481610 for now
